docs/ChainOrderTransitivity.md

```diff
@@ -0,0 +1,153 @@
+# Transitivity of the chain order
+
+## Context
+
+The abstract Consensus layer currently implements the chain order like this:
+From every header, one can extract a `SelectView`, which has an `Ord` instance.
+A chain `C` is _preferable_ to a chain `C'` if `selectView(C) > selectView(C')` where `C` and `C'` are the tip headers of `C` and `C'`, respectively.
+
+## Examples for intransitive orders
+
+### Current chain order is not transitive
+
+For all Shelley-based eras, we currently use a lexicographic combination of the following comparisons in descending order:
+
+ 1. Chain length, preferring longer chains.
+ 2. If the issuer identity is the same, compare by the issue/opcert number, preferring higher values, otherwise, no preference.
+ 3. VRF tiebreaker[^vrf-tpraos-vs-praos], preferring lower values.
+
+This is not a total order: It is is reflexive, antisymmetric and total, but not transitive.
```

Comment:
is is

Reply:
Odd to say it's not a "total order" and then call it "total". Maybe writing "the relation is reflexive..." instead of "it is reflexive..." would help.